### PR TITLE
N18: added dwarf-world

### PR DESCRIPTION
**Pull request by AlexEne**
I've added dwarf world.

[contributing]: https://github.com/rust-gamedev/rust-gamedev.github.io/blob/source/CONTRIBUTING.md

### CODE REVIEW

File: content/posts/newsletter-018/index.md [hunk 1]
```diff
@@ -230,6 +230,31 @@ which will be released in the Steam.
 [Station Iapetus]: https://github.com/mrDIMAS/StationIapetus
 [si-youtube]: https://www.youtube.com/watch?v=JCH2U5JOMlU
 
+### [Dwarf world][dwarf-world]
+
+![Dwarf world](dwarfworld_lights.gif)
+
+[Dwarf World] by [Alex Ene][@_AlexEne_] has added a couple of updates and
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-018/index.md:237]: The URLs for these links are missing.

Comment (AlexEne) [reply content/posts/newsletter-018/index.md:237]: They are mentioned above in the podcast section and I thought it's not needed to add them again

Comment (17cupsofcoffee) [reply content/posts/newsletter-018/index.md:237]: Oh sorry, you're totally right - one of the links seems to be broken, but I can fix that via GitHub UI :+1:

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by 17cupsofcoffee** [comment]
Thank you for the PR :)

**Pull request merged**